src/test/resources/test-exception-3.kore

```diff
@@ -1,8 +0,0 @@
-[]
-
-module TEST-EXCEPTION-3
-
-  axiom{S}
-    'X:S []
```

Comment:
Do we have an equivalent test with strings?

Reply:
good catch, we didn't.